versions/1.1.0-dev.md

```diff
@@ -254,7 +277,7 @@ info:
   title: Apply Traits
   version: 1.0.0
 actions:
-  - target: $.paths.*.get[?@.x-oai-traits.paged]
+  - target: $.paths.*.get[?(@.x-oai-traits[?(@ == 'paged')])]
```

Comment:
leaving a note to check at a later time whether it's fully compliant with RFC 9535 since both JsonPath.net and https://jsonpath.com/ complain about this one.